{{cookiecutter.project_name}}/tasks.py

```diff
@@ -200,59 +237,71 @@ def pre_commit(c: Context, auto_fix: bool):
     if ("fixed" in result.stdout or "reformatted" in result.stdout) and auto_fix:
         _add_commit(c, msg="style: Auto-fixes from pre-commit")
 
-        print(f"{Msg.DOING} Fixed errors, re-running pre-commit checks")
+        print(f"{msg_type.DOING} Fixed errors, re-running pre-commit checks")
         second_result = c.run(pre_commit_cmd, pty=NOT_WINDOWS, warn=True)
         exit_if_remaining_errors(second_result)
     else:
         if result.return_code != 0:
-            print(f"{Msg.FAIL} Pre-commit checks failed")
+            print(f"{msg_type.FAIL} Pre-commit checks failed")
             exit(1)
 
 
-def mypy(c: Context):
-    echo_header(f"{Msg.CLEAN} Running mypy")
-    c.run("mypy .", pty=NOT_WINDOWS)
+@task
+def static_type_checks(c: Context):
+    echo_header(f"{msg_type.CLEAN} Running static type checks")
+    c.run("pyright .", pty=NOT_WINDOWS)
```

Comment:
Since we already have the `NOT_WINDOWS` constant, we can reuse that for the MSG_TYPE instead of using its custom method.